src/App/FinancialPositions/sagas.ts

```diff
@@ -127,13 +125,8 @@ function* updateFinancialPositionsParticipant() {
   assert(updateAmount !== 0, 'Value 0 is not valid for Amount');
 
   const position: FinancialPosition = yield select(getSelectedFinancialPosition);
-  const accounts = yield call(apis.accounts.read, { dfspName: position.dfsp.name });
-  assert(accounts.status === 200, 'Unable to fetch DFSP data');
-
-  const account = accounts.data.filter(
-    (acc: { ledgerAccountType: string }) => acc.ledgerAccountType === 'SETTLEMENT',
-  )[0];
 
+  const account = position.settlementAccount;
```

Comment:
I think we don't need to get the accounts again here as we already have account object. Please correct me if this is intended to serve some other use-case.
I am doing this because its always picking up the first SETTLEMENT account if a DFSP has multiple currencies